soh/soh/Enhancements/presets.cpp

```diff
@@ -12,6 +13,14 @@ void clearCvars(std::vector<const char*> cvarsToClear) {
     }
 }
 
+std::string FormatLocations(std::vector<RandomizerCheck> locs) {
+    std::string locString = "";
+    for (auto loc: locs) {
+        locString += std::format("{},", (uint32_t)loc);
```

Comment:
Since we support C++11 you can just do string concat with `std::to_string`, we already do this in many places.

Reply:
You mean like `locString += std::to_string(loc) + ",";`?